pkg/util/utils.go

```diff
@@ -210,3 +211,13 @@ func ParseIDMapping(UIDMapSlice, GIDMapSlice []string, subUIDMap, subGIDMap stri
 	}
 	return &options, nil
 }
+
+// GetAuthFile gets the value of --authfile option or REGISTRY_AUTH_FILE variable
+// "" if not found
+func GetAuthFile(c *cli.Context) string {
+	authfile := c.String("authfile")
+	if authfile == "" {
+		authfile = os.Getenv("REGISTRY_AUTH_FILE")
```

Comment:
nit: return os.Getenv("REGISTRY_AUTH_FILE")

Reply:
This should go into common.go or utils.go under cmd/podman not here.  No reason to hard code CLI functions into this global package.

Reply:
Maybe cmd/podman/shared is a good place to land it, because I can see this being used in Varlink code

Reply:
Oh, I missed the location, yes, please move this in your next PR.  cmd/podman/shared sounds good to me.

Reply:
Add a new file named ```utils.go``` under cmd/podman/shared or add this function to ```funcs.go```?

Reply:
Well cli is not handled there either, so I am not sure it is best there.  
I am not sure this function is even necessary.

Reply:
If we don't make the function, we'll need to add the 3 or 4 lines of code in the function to a half dozen or so commands.  I just thought it was cleaner to do the one line call to the function in each, especially so if varlink can also make use of it.

I'll let @mheon decide and note a location if appropriate.

Reply:
Remove the CLI dependency (make it take a string and pass in `c.String("authfile")`) and it'll be a good fit for `cmd/podman/common`

Reply:
SGTM
```
func GetAuthFile(authfile string) string {
	if authfile != "" {
               return authfile
        }
        return os.Getenv("REGISTRY_AUTH_FILE")
}
```